src/pages/Proposal/VoteModal/index.tsx

```diff
@@ -85,10 +87,16 @@ const VoteModal: React.FC<IVoteModal> = ({
             <FormikTextField
               label={t('vote.comment')}
               name="reason"
+              id="reason"
               placeholder={t('vote.addComment')}
               css={styles.comment}
             />
-            <FormikSubmitButton enabledLabel={title} fullWidth loading={isVoteLoading} />
+            <FormikSubmitButton
+              enabledLabel={title}
+              fullWidth
+              loading={isVoteLoading}
+              data-testid={TEST_IDS.voteModal.submitButton}
```

Comment:
In this instance we can use the content directly to target the button since it doesn't include any string interpolation

Reply:
This is a valid solution. It is better than using the translation string because the text appears multiple times on the page. Using a testId allows the selection of the specific component we want to use

Reply:
This is a valid solution. It is better than using the translation string because the text appears multiple times on the page. Using a testId allows the selection of the specific component we want to use